### ISSUE TITLE
[Suggestion] Translations for UI/Game text

### ISSUE THREAD

**Issue by Blueberryy** [nzp-team/nzportable#95]
Nazi Zombies is amazing thing can be played anywhere in any point of world. I think translations can be handled by community

**Comment by Paindeluxe** [comment]
I agree especially since a suprising amount of homebrew titles have more than 1 Language available so why not NZP aswell?

**Comment by BCDeshiG** [comment]
From what I understand, we're currently limited to the basic ASCII characters seen in the [`conchars.tga`](https://github.com/BCDeshiG/assets/blob/main/pc/nzp/gfx/conchars.tga?raw=true) file as shown here:
![conchars](https://user-images.githubusercontent.com/29164668/210110450-4cf866cb-02c1-4eac-b655-8ab4cba6f011.png)

**Comment by Blueberryy** [comment]
> From what I understand, we're currently limited to the basic ASCII characters seen in the [`conchars.tga`](https://github.com/BCDeshiG/assets/blob/main/pc/nzp/gfx/conchars.tga) file as shown here:
> ![conchars](https://user-images.githubusercontent.com/29164668/210110450-4cf866cb-02c1-4eac-b655-8ab4cba6f011.png)
> 

So we need to get some artist to draw extra symbols and update the file?

**Comment by MotoLegacy** [comment]
Hi! Sorry for taking so long to get around to this. I would love to add some sort of translation support but there are a lot of obstacles in the way.

The first one is, as @BCDeshiG stated, currently the game is limited by standard ASCII English characters, and each character is limited to an 8x8 space. While we could have an artist to draw more, and implement support for loading the additional character map, the 8x8 space for characters is definitely a limiting factor for languages that are logographic, for example.

The next obstacle is the complications it would create for map creators wanting to create custom strings. We would have to either size up our entity fields considerably to accommodate every supported language or add support for localized strings, either option complicating the process or making language features feel incomplete.

I'm not going to close this issue, as I feel this at it's core is a huge accessibility opportunity. Instead I'm going to mark it as help wanted to see if anyone can come along offering better implementation ideas and actual implementation examples to integrate into our source ports.
